modules/post/windows/gather/avast_memory_dump.rb

```diff
@@ -31,7 +31,19 @@ def initialize(info = {})
   end
 
   def avdump_exists?
-    file_exist?('C:\\Program Files\\Avast Software\\Avast\\AvDump.exe')
+    avdump_paths = [
+      'C:\\Program Files\\Avast Software\\Avast\\AvDump.exe',
+      'C:\\Program Files\\Avast Software\\BreachGuard\\AvDump.exe',
+      'C:\\Program Files\\Avast Software\\Cleanup\\AvDump.exe',
+      'C:\\Program Files\\Avast Software\\Driver Updater\\AvDump.exe',
+      'C:\\Program Files\\Avast Software\\SecureLine VPN\\AvDump.exe'
+    ]
+
+    avdump_paths.each do |p|
+      if file_exist?(p.to_s)
+        return p.to_s
+      end
+    end
```

Comment:
In addition, I think converting these to use a path substitution to use the target system's program files directory would be useful while we're at it
```suggestion
    avdump_paths = [
      'Avast\\AvDump.exe',
      'BreachGuard\\AvDump.exe',
      'Cleanup\\AvDump.exe',
      'Driver Updater\\AvDump.exe',
      'SecureLine VPN\\AvDump.exe'
    ]

    base = expand_path('%PROGRAMFILES%\\Avast Software\\')
    avdump_paths.each do |p|
      if file_exist?(base + p.to_s))
        return base + p.to_s
      end
    end
```

Reply:
> In addition, I think converting these to use a path substitution to use the target system's program files directory would be useful while we're at it
> 
> ```suggestion
> 
>     avdump_paths = [
> 
>       'Avast\\AvDump.exe',
> 
>       'BreachGuard\\AvDump.exe',
> 
>       'Cleanup\\AvDump.exe',
> 
>       'Driver Updater\\AvDump.exe',
> 
>       'SecureLine VPN\\AvDump.exe'
> 
>     ]
> 
> 
> 
>     base = expand_path('%PROGRAMFILES%\\Avast Software\\')
> 
>     avdump_paths.each do |p|
> 
>       if file_exist?(base + p.to_s))
> 
>         return base + p.to_s
> 
>       end
> 
>     end
> 
> ```

Great point! I'll add and push later this week/weekend.

Reply:
Suggestion solution from Adam has been implemented, so going to mark this as resolved.